Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -548,6 +548,11 @@ void TerminalWidget::set_opacity(u8 new_opacity)
     update();
 }
 
+void TerminalWidget::set_show_scrollbar(bool show_scrollbar)
```

Comment:
I noticed a typo in the commit message, maybe it should say:

> In preparation for another feature, expose an API so that any users of this widget can control the scrollbar visibility.

Reply:
Thanks!